### DISCUSSION TITLE
SmartWings Shade Position Tracking Issues

### DISCUSSION THREAD

**Discussion by djw1191** [zwave-js/zwave-js#7049] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

I've got a Smartwings branded top down/bottom up shade that is having issues with consistent reporting of level. First reported over in the HA github, but the initial response over there is that it is likely a zwavejs issue and that it may be similar to #5975 

The issue is observed when starting a level change, current level of the shade is not reported in the Window Covering and instead the current level is reported in the multilevel switch CC. When setting an explicit level, the correct level does appear to be reported in the CC106, in addition to CC38. I think as a result of the multilevel switch CC, a light is also reported inside of home assistant.

I've attached a debug log, that captures both the re-interview and a level change showing how the device reports the position.

https://github.com/home-assistant/core/issues/122681

### Software versions

zwave-js-ui: 9.16.3.7d47bd2
zwave-js: 13.0.2

Home Assistant Z-Wave Integration: 2024.7.3

### Device information

Manufacturer: 0x045a
Model name: 0x0004 0x0509
Node ID: 002

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_2024-07-27 (1).log](https://github.com/user-attachments/files/16397899/zwavejs_2024-07-27.1.log)

**Comment by brononius** [comment]
Did you got a solution/workaround for this? I'm encountering the same issue...

**Reply by djw1191** [reply]
> Did you got a solution/workaround for this? I'm encountering the same issue...

I did not, but I did find doing a zwave refresh from home assistant would at least sync the values, so I did do that as a workaround via an automation.

**Reply by brononius** [reply]
Ah, got that part. 
Just, how do you trigger this refresh? Is this timebased, or something else? 
I can't find a trigger that will refresh it when I push for example the remote.

**Comment by AlCalzone** [comment]
Somehow I missed this. Will take a look tomorrow.

**Reply by brononius** [reply]
OK, let us know if we can provide/test/... anything!

ps bit anoying that when you lower the shades with the 'down-button' or the remote, the status is still 'open'. And since the 'up-botton' in HA is disabled, no way to open it again.

**Comment by AlCalzone** [comment]
Seems to be the same issue as https://github.com/zwave-js/node-zwave-js/issues/6318
